src/Adapter/LegacyContext.php

```diff
@@ -140,6 +160,18 @@ public function getRootUrl()
         return __PS_BASE_URI__;
     }
 
+    /**
+     * Url to the mail themes folder
+     *
+     * @since 1.7.6
+     *
+     * @return string
+     */
+    public function getMailThemesUrl()
+    {
+        return $this->tools->getShopDomain(true) . __PS_BASE_URI__ . $this->mailThemesUri;
```

Comment:
getShopDomain is static.

Reply:
You're right in the `PrestaShop\PrestaShop\Adapter\Tool` it needs to be a classic public function